src/main/java/com/minecolonies/commands/ColoniesCommand.java

```diff
@@ -11,23 +17,50 @@
  */
 public class ColoniesCommand extends AbstractSplitCommand
 {
+    public static final String DESC = "colonies";
 
     private final ImmutableMap<String, ISubCommand> subCommands =
       new ImmutableMap.Builder<String, ISubCommand>()
-        .put("list", new ListColonies("mc", "colonies", "list"))
+        .put("list", new ListColonies("minecolonies", DESC, "list"))
         .build();
 
     /**
      * Initialize this command with it's parents.
      */
-    public ColoniesCommand()
+    public ColoniesCommand(@NotNull final String parent)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): parent [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)